modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -2264,6 +2266,19 @@ void testStartBuildingIndexEvent() {
         assertThat(fireEventFuture, willCompleteSuccessfully());
     }
 
+    @Test
+    void testCatalogVersionsSnapshot() {
+        createSomeTable(TABLE_NAME);
+        createSomeIndex(TABLE_NAME, INDEX_NAME);
+
+        List<Catalog> expCatalogs = IntStream.rangeClosed(manager.earliestCatalogVersion(), manager.latestCatalogVersion())
+                .mapToObj(manager::catalog)
+                .collect(toList());
+
+        assertThat(expCatalogs, hasSize(greaterThanOrEqualTo(2)));
```

Comment:
Since there were two catalog changes, I check that the version must be greater than 2. 
Why is it not equal, because at the start of the `CatalogManagerImpl` there is immediately an empty version and in the end there are 3 versions.